### PR TITLE
GUI fixes, sh file for run automation

### PR DESCRIPTION
**Pull request by mprtcz**
For reviewers: 
In the pull request there are some math methods as well as collecting methods which will need to be checked;
Any suggestions abou the sh file are also very welcome.

### CODE REVIEW

File: RunServerAndTwoClients.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mvn exec:java -Dexec.mainClass="com.spanish_inquisition.battleship.server.BattleshipServer" &
+mvn exec:java -Dexec.mainClass="com.spanish_inquisition.battleship.client.Client" &
+mvn exec:java -Dexec.mainClass="com.spanish_inquisition.battleship.client.Client" &
```
Comment (patkkka) [inline RunServerAndTwoClients.sh]: 👍

File: src/main/java/com/spanish_inquisition/battleship/common/Header.java [hunk 2]
```diff
@@ -25,7 +25,7 @@ public static Header parseResponseHeader(String responseHeader) {
         try {
             header = valueOf(responseHeader);
         } catch (IllegalArgumentException e) {
-            logger.log(Level.WARNING, e.getMessage(), e);
+            logger.log(Level.WARNING, e.getMessage());
```
Comment (patkkka) [inline src/main/java/com/spanish_inquisition/battleship/common/Header.java]: :godmode:

File: RunServerAndTwoClients.sh (hunk 1 above)
Comment (kukulastudent) [inline RunServerAndTwoClients.sh]: :+1:

File: src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
 /**
  * @author Michal_Partacz
  * This class will make sure the board of buttons is properly initialized and with every event handlers needed.
  */
 public class GameBoardBuilder {
-    private static final int BOARD_SIZE_WITH_LABELS = 11;
-    private static final double FIELD_SIZE = 10;
+    public static final int BOARD_SIZE_WITH_LABELS = 11;
+    public static final double FIELD_SIZE = 30;
     private GridPane gridPane;
```
Comment (kukulastudent) [inline src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java]: :+1:

File: src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java [hunk 4]
```diff
@@ -57,23 +76,32 @@ private void createNewBoardTileAndAddIt(int absoluteXCoordinate, int absoluteYCo
         fillGridPaneHeightAndWidthForNode(tile);
     }
 
-    private void addVerticalLabelOnIndex(int columnIndex) {
-        Label boardBoardTileLabel = new Label(String.valueOf(columnIndex));
-        Platform.runLater(() -> this.gridPane.add(boardBoardTileLabel, 0, columnIndex));
+    private void addHorizontalLabelOnIndex(int rowIndex) {
+        Label boardBoardTileLabel = buildCenteredLabelWithText(String.valueOf(rowIndex));
+        Platform.runLater(() -> this.gridPane.add(boardBoardTileLabel, 0, rowIndex));
```
Comment (kukulastudent) [inline src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java]: :+1: lambdas

File: src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java [hunk 5]
```diff
@@ -82,4 +110,9 @@ public void handle(MouseEvent event) {
             }
```
Comment (kukulastudent) [inline src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java]: empty?

File: src/test/java/com/spanish_inquisition/battleship/client/network/SocketClientTest.java [hunk 6]
```diff
@@ -8,8 +8,7 @@
 import java.net.Socket;
 import java.nio.charset.StandardCharsets;
 
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 import static org.testng.Assert.*;
```
Comment (kukulastudent) [inline src/test/java/com/spanish_inquisition/battleship/client/network/SocketClientTest.java]: why import all ?

**Pull request merged**